### PR TITLE
Add doc for Management Server Statistics and Peers

### PR DESCRIPTION
**Pull request by weizhouapache**
for apache/cloudstack#9885

----
📚 Documentation preview 📚: https://cloudstack-documentation--455.org.readthedocs.build/en/455/

### CODE REVIEW

**Comment by weizhouapache** [comment]
new page:
https://cloudstack-documentation--455.org.readthedocs.build/en/455/adminguide/management.html#management-server-statistics-and-peers

File: source/adminguide/management.rst [hunk 1]
```diff
@@ -587,3 +587,52 @@ To start the Management Server:
...
+|management-servers-list.png|
+
+#. Click the management server you'd like to view. The statistics of the management server are displayed.
+
+|management-server-statistics.png|
+
+#. Navigate to the "Peers" tab. The peers of the management servers are listed
+
+|management-server-peers.png|
+
```
Comment (sureshanaparti) [inline source/adminguide/management.rst:611]: @weizhouapache can rename the ms and peer names, and update the screenshots?

**Comment by borisstoyanov** [comment]
@blueorangutan docbuild

File: source/adminguide/management.rst (hunk 1 above)
Comment (weizhouapache) [reply source/adminguide/management.rst:611]: will do

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

File: source/adminguide/management.rst (hunk 1 above)
Comment (weizhouapache) [reply source/adminguide/management.rst:611]: updated

**Review by borisstoyanov — APPROVED** [review]
LGTM

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**